src/modules/reading-room/components/forms/AdvancedFilterForm/AdvancedFilterFields/AdvancedFilterFields.types.ts

```diff
@@ -0,0 +1,21 @@
+import { AdvancedFilterFieldsState } from '../AdvancedFilterForm.types';
+
+export interface AdvancedFilterFieldsProps {
+	id: string;
+	index: number;
+	value: AdvancedFilterFieldsState;
+	onChange: (index: number, values: Partial<AdvancedFilterFieldsState>) => void;
+	onRemove: (index: number) => void;
+}
+
+export enum MetadataProp {
+	Title = 'title',
+	SecundaryTitle = 'secundary_title',
```

Comment:
@bertyhell / @bartnaessens is deze `secund...`-typo ergens anders van toepassing? Komt deze bijvoorbeeld terug in elastic of ergens in de proxy?

Reply:
nee, ik heb de typo gefixed